### PR TITLE
Fix broken md linting in schema submodule

### PR DESCRIPTION
**Pull request by hickeyma**
**Description of your changes:**

Pulls the latest schema commit whereby the schema linting has been fixed. Also, disables linting of schema in this repo as it should be linted in its own repo.

### CODE REVIEW

**Review by bjhargrave — CHANGES_REQUESTED** [review]
This should really be a change to 

https://github.com/instructlab/instructlab/blob/a4afed344d0916802da6d1a5985abc9424579b1b/.markdownlint-cli2.yaml#L14

to add `src/instructlab/schema/**` to the ignore list.

This repo's linting should not be linting the submodules. I will add a markdown linting option to the schema repo which will include any needed md fixes.

**Comment by bjhargrave** [comment]
In taxonomy repo, we exclude the schema submodule from linting.

https://github.com/instructlab/taxonomy/blob/7ce23451b9e8598567029360d78da103718b6837/.markdownlint-cli2.yaml#L14

**Comment by hickeyma** [comment]
Thanks for the review @bjhargrave. Updated and ready for review again.

File: .markdownlint-cli2.yaml [hunk 1]
```diff
@@ -13,7 +13,8 @@ globs:
   - "**/*.md"
 ignores:
   - ".tox/**"
-  - "venv/**"
   - ".venv/**"
   - "build/**"
+  - "schema/**"
```
Comment (bjhargrave) [inline .markdownlint-cli2.yaml:18]:
```suggestion
  - "src/instructlab/schema/**"
```

This needs to use the folder of the schema submodule in this repo.

**Review by bjhargrave — CHANGES_REQUESTED** [review]

File: .markdownlint-cli2.yaml (hunk 1 above)
Comment (hickeyma) [reply .markdownlint-cli2.yaml]: Doh, copy/paste! :-D

**Comment by russellb** [comment]
@bjhargrave can you take another look?

**Review by bjhargrave — APPROVED** [review]

**Pull request merged**